connect_ext_ppr/utils.py

```diff
@@ -316,6 +317,53 @@ def get_configuration_schema(configuration, file):
     )
 
 
+def get_ppr_version_schema(ppr, file, conf):
+    """
+    Returns PPRVersionSchema for the PPR
+    :param ppr: PPRVersion model
+    :param file: File model
+    :param conf: Related Configuration model
+    :rtype: PPRVersionSchema
+    """
+    file_schema = FileSchema(
+        id=file.id,
+        name=file.name,
+        location=file.location,
+        size=file.size,
+        mime_type=file.mime_type,
+    )
+
+    conf_schema = None
+    if conf:
+        conf_schema = ConfigurationReferenceSchema(
+            id=conf.id,
+            state=conf.state,
+        )
+
+    description_items = []
+    if ppr.description:
+        description_items.append(ppr.description)
+    if ppr.summary:
+        description_items.append(str(ppr.summary))
+    description = '\n'.join(description_items)
```

Comment:
Better to decide, how to output description and summary. Now I merge them because there are case when two of them are present and make sense. Like upload file with a description, but facing the validation errors.

Reply:
I have new task to implement this, but for now is good to have at least this approach